Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -18,27 +18,11 @@ extension FileManager: LintableFileManager {
         let absolutePath = path.bridge()
             .absolutePathRepresentation(rootDirectory: rootPath).bridge()
             .standardizingPath
-        var isDirectoryObjC: ObjCBool = false
-        guard fileExists(atPath: absolutePath, isDirectory: &isDirectoryObjC) else {
-            return []
-        }
-        #if os(Linux)
-        let isDirectory = isDirectoryObjC
-        #else
-        let isDirectory = isDirectoryObjC.boolValue
-        #endif
-        if isDirectory {
-            do {
-                return try subpathsOfDirectory(atPath: absolutePath)
-                    .map(absolutePath.bridge().appendingPathComponent).filter {
-                        $0.bridge().isSwiftFile()
-                    }
-            } catch {
-                fatalError("Couldn't find files in \(absolutePath): \(error)")
+        return enumerator(atPath: absolutePath)?.flatMap { element in
+            if let element = element as? String, element.hasSuffix(".swift") {
```

Comment:
Sure. I didn't do it before to avoid the `bridge()`, but might as well use the method we have for this.